### PR TITLE
Added template build

### PR DESCRIPTION
**Pull request by JKapella**
Created template for build - wireframe available here - https://docs.google.com/presentation/d/1UslObd5AaJhkfURtPAbaPRupwlq5gcKeBdOHiKviUY0/edit?usp=sharing

You don't need to review the code that i've started on the 'about me' section. Also, I will be removing the lorem ipsum, from the portfolio section, so this doesn't need to be accounted for. 

Thanks,

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+
+html {
+    background-image: url('img/backgroundImage.jpg');
+    background-color: #000000;
+    background-size: 1920px 1080px;
+    background-repeat: no-repeat;
+    background-attachment: fixed;
+
+
+    -moz-osx-font-smoothing: grayscale;
```
Comment (aldofiore28) [inline styles.css]: This line of code does not look like to make any difference to the text in the texts.

File: styles.css [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+
+.container {
+    width: 650px;
+    margin: 0 auto;
+}
+
+.scrollingFingerImage {
+    display: none;
+}
+
```
Comment (aldofiore28) [inline styles.css]: The text overlap the header before 1270px

File: styles.css (hunk 1 above)
Comment (aldofiore28) [reply styles.css]: *the way the text looks

Comment (aldofiore28) [inline styles.css]: This line does not do anything.

File: styles.css (hunk 2 above)
Comment (aldofiore28) [inline styles.css:82]: The main elements overlap before 1270px

**Review by aldofiore28 — COMMENTED** [review]
It looks good.

File: styles.css (hunk 2 above)
Comment (JKapella) [reply styles.css:82]: Magnificent, thanks

File: styles.css (hunk 1 above)
Comment (JKapella) [reply styles.css]: Yeah, that will only be on firefox!

**Review by aldofiore28 — APPROVED** [review]
It's good.

**Pull request merged**